### Aufgabe 12b:
Löse die folgenden Aufgaben in Python und speichere das Ergebnis unter *VornameNachname*-aufgabe12b.ipynb auf Laufwerk H:.

1. Finde alle Zahlen zwischen 0 und 1000, die durch 47 teilbar sind.

#### Hinweis:

Wenn man eine Komma-Zahl mit `int(zahl)` in eine ganze Zahl umwandelt und von `zahl` abzieht, bleiben nur die Nachkommastellen übrig. 
Wenn eine Division aufgeht, hat das Ergebnis der Division keine Nachkommastellen (d. h. die sind alle Null).

In [20]:
Teiler = 47
for n in range(1000):
    zahl = n / float(Teiler)
    if int(zahl)-zahl == 0:
        print( n,"ist",int(zahl),"mal",Teiler )

0 ist 0 mal 47
47 ist 1 mal 47
94 ist 2 mal 47
141 ist 3 mal 47
188 ist 4 mal 47
235 ist 5 mal 47
282 ist 6 mal 47
329 ist 7 mal 47
376 ist 8 mal 47
423 ist 9 mal 47
470 ist 10 mal 47
517 ist 11 mal 47
564 ist 12 mal 47
611 ist 13 mal 47
658 ist 14 mal 47
705 ist 15 mal 47
752 ist 16 mal 47
799 ist 17 mal 47
846 ist 18 mal 47
893 ist 19 mal 47
940 ist 20 mal 47
987 ist 21 mal 47


### Lösung:

0 ist 0 mal 47<br>
47 ist 1 mal 47<br>
94 ist 2 mal 47<br>
141 ist 3 mal 47<br>
188 ist 4 mal 47<br>
235 ist 5 mal 47<br>
282 ist 6 mal 47<br>
329 ist 7 mal 47<br>
376 ist 8 mal 47<br>
423 ist 9 mal 47<br>
470 ist 10 mal 47<br>
517 ist 11 mal 47<br>
564 ist 12 mal 47<br>
611 ist 13 mal 47<br>
658 ist 14 mal 47<br>
705 ist 15 mal 47<br>
752 ist 16 mal 47<br>
799 ist 17 mal 47<br>
846 ist 18 mal 47<br>
893 ist 19 mal 47<br>
940 ist 20 mal 47<br>
987 ist 21 mal 47<br>